src/Symfony/Component/Messenger/Tests/Command/FailedMessagesShowCommandTest.php

```diff
@@ -94,4 +96,107 @@ public function testMultipleRedeliveryFails()
             $redeliveryStamp2->getRedeliveredAt()->format('Y-m-d H:i:s')),
             $tester->getDisplay(true));
     }
+
+    public function testRecevierShouldBeListable()
+    {
+        $receiver = $this->createMock(ReceiverInterface::class);
+        try {
+            $command = new FailedMessagesShowCommand(
+                'failure_receiver',
+                $receiver
+            );
+            $tester = new CommandTester($command);
+            $tester->execute(['id' => 15]);
+
+            $this->fail(sprintf('Expected %s to be thrown', RuntimeException::class));
+        } catch (RuntimeException $expectedException) {
+            $this->assertEquals('The "failure_receiver" receiver does not support listing or showing specific messages.', $expectedException->getMessage());
```

Comment:
You should use `$this->expectException` & `$this->expectExceptionMessage` instead of try/catch + fail

Reply:
I got it from another test. but your suggestion is better :+1 Thank you